Connectors/Rhino/Speckle.Connectors.RhinoShared/HostApp/RhinoLayerBaker.cs

```diff
@@ -123,6 +123,12 @@ private int CreateLayerFromPath(Collection[] collectionPath, string baseLayerNam
       }
 
       var cleanNewLayerName = RhinoUtils.CleanLayerName(collection.name);
+
+      if (!ModelComponent.IsValidComponentName(cleanNewLayerName))
+      {
+        throw new SpeckleException($"Layer name '{currentLayerName}' is not valid");
+      }
```

Comment:
if some other gremlin rears its head, we will at least see more informative logs:

```
Speckle.Sdk.SpeckleException: Layer name 'Project REVIT MODELS CONFIDENTIAL: Model a-0040-v0001 (1)::A-0040-V0001 1::A-0040-V0001 1::PLAN 1::Walls::YV_N 200 Isolering' is not valid
```

As opposed to:

```
System.InvalidOperationException: ModelComponent.set_Id failed. The component is likely locked.
```